### DISCUSSION TITLE
How to Achieve More Achievements？

### DISCUSSION THREAD

**Discussion by rickie1219** [MALTF/APK-Parser#5] [Q&A] [answered]
How to Achieve More Achievements, brother？

**Comment by MALTF** [comment, marked as answer]
GitHub Profile Achievements 🏆
A collection listing all Achievements available on the GitHub profile.

Following the launch of the first Mars Helicopter, [Ingenuity](https://en.wikipedia.org/wiki/Ingenuity_(helicopter)), GitHub [announced](https://github.blog/2021-04-19-open-source-goes-to-mars/) the new Achievements section:

We are also using this opportunity to introduce a new Achievements section to the GitHub profile. Right now, Achievements include the Mars 2020 Helicopter Mission badge, the [Arctic Code Vault](https://archiveprogram.github.com/arctic-vault/) badge, and a badge for sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors).

This repository attempts to list them all.

2022-06-09: GitHub [announced more Achievements](https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/), but without specific criteria. If you see something not listed here or is incorrect, please open an issue :)

Don't like the Achievements? [Here](https://docs.github.com/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements) is how to disable them. [Direct link to the setting for your convenience](https://github.com/settings/profile#profile-settings-heading).

**Comment by MALTF** [comment]
How to get the YOLO badge？

- [x] it can be your own repo

- [x] it can be only one line changed

- [x] default commit message, default PR title and empty description is possible the achievement is awarded instantly I merged using the usual "Merge pull request" with default option "Create a merge commit" selected (not sure about the other options)

- [x] For my PR: At the time of the merge, the reviewer was a collaborator of the repo (previously invited and accepted, so I could request a review)
